src/com/jwetherell/algorithms/mathematics/Division.java

```diff
@@ -22,6 +31,10 @@ public static final long divisionUsingLoop(int a, int b) {
     }
 
     public static final long divisionUsingRecursion(int a, int b) {
+        // Error if divide by zero
+        if (b == 0) {
```

Comment:
This will check if the divisor is zero in this method. If this is true, a IllegalArgumentException will be thrown with the error message "/ by zero". This will help in identifying bugs when the divisor is set to 0 unintentionally. It also will prevent unexpected crashes or behaviour.